### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by bednarz1990**

### CODE REVIEW

File: Apidaze.SDK.Tests.Integration/DownloadRecordingToFileExample/Program.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ static void Main()
             }
 
             // initialize API factory
-            var apiFactory = ApplicationManager.CreateApiFactory(new Credentials(apiKey, apiSecret));
+            var apiFactory = ApplicationManager.CreateApiFactory(new Credentials(apiKey, apiSecret), "https://api.apidaze.io");
```
Comment (shadzik) [inline Apidaze.SDK.Tests.Integration/DownloadRecordingToFileExample/Program.cs]: Do you have to specify the URL every time? Could it have a default value of `https://api.apidaze.io`?

File: Apidaze.SDK.Tests.Integration/IvrExample/WebService.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ internal static class WebService
         /// <summary>
         /// The port the HttpListener should listen on.
         /// </summary>
-        private static readonly string SERVER_URL = "http://";
+        private static readonly string SERVER_URL = "http://.ngrok.io";
```
Comment (shadzik) [inline Apidaze.SDK.Tests.Integration/IvrExample/WebService.cs]: No ads for ngrok.io please, we are in no way affiliated with them

File: Apidaze.SDK.Tests.Integration/UpdateExternalScriptUrl/Program.cs [hunk 3]
```diff
@@ -35,14 +35,14 @@ static void Main(string[] args)
             var id = 1801L;
 
             // new url
-            var newScriptUrl = "http://";
+            var newScriptUrl = "http://.ngrok.io";
```
Comment (shadzik) [inline Apidaze.SDK.Tests.Integration/UpdateExternalScriptUrl/Program.cs]: Same here, no ngrok please. Please check if your code has reference to ngrok somewhere

**Review by shadzik — CHANGES_REQUESTED** [review]

File: Apidaze.SDK.Tests.Integration/DownloadRecordingToFileExample/Program.cs (hunk 1 above)
Comment (bednarz1990) [reply Apidaze.SDK.Tests.Integration/DownloadRecordingToFileExample/Program.cs]: No, I don't have. I will remove this one.

File: Apidaze.SDK.Tests.Integration/IvrExample/WebService.cs (hunk 2 above)
Comment (bednarz1990) [reply Apidaze.SDK.Tests.Integration/IvrExample/WebService.cs]: ok

File: Apidaze.SDK.Tests.Integration/UpdateExternalScriptUrl/Program.cs (hunk 3 above)
Comment (bednarz1990) [reply Apidaze.SDK.Tests.Integration/UpdateExternalScriptUrl/Program.cs]: ok.

**Review by barinali — APPROVED** [review]

**Review by shadzik — APPROVED** [review]

**Pull request merged**